appservice/src/ISimplifiedSiteClient.ts

```diff
@@ -4,15 +4,20 @@
 *--------------------------------------------------------------------------------------------*/
 
 import { SiteConfigResource, SiteSourceControl, User } from 'azure-arm-website/lib/models';
-import { KuduClient } from 'vscode-azurekudu';
-
-export interface IDeploymentsClient {
+import KuduClient from 'vscode-azurekudu';
+import { IAppSettingsClient } from './IAppSettingsClient';
+import { IHostKeys } from './SiteClient';
 
+export interface ISimplifiedSiteClient extends IAppSettingsClient {
     fullName: string;
```

Comment:
fullName and isLinux are both part of IAppSettingsClient, so you can remove that from this interface